f5/bigip/tm/sys/management_route.py

```diff
@@ -0,0 +1,50 @@
+# coding=utf-8
+#
+# Copyright 2016 F5 Networks Inc.
```

Comment:
we dont need to update all source files in the source tree for copyright, just the new files that we add.